arrow-data/src/main/kotlin/arrow/data/Try.kt

```diff
@@ -240,15 +246,71 @@ sealed class Try<out A> : TryKind<A> {
                 }
             }
 
+    abstract fun isFailure(): Boolean
+
+    abstract fun isSuccess(): Boolean
+
+    @Deprecated(DeprecatedUnsafeAccess, ReplaceWith("fold({ Unit }, f)"))
+    fun foreach(f: (A) -> Unit) {
+        if (isSuccess()) f(get())
+    }
+
+    @Deprecated(DeprecatedUnsafeAccess, ReplaceWith("map { f(it); it }"))
+    fun onEach(f: (A) -> Unit): Try<A> = map {
+        f(it)
+        it
+    }
+
+    fun exists(predicate: Predicate<A>): Boolean = fold({ false }, { predicate(it) })
+
+    @Deprecated(DeprecatedUnsafeAccess, ReplaceWith("getOrElse { ifEmpty }"))
+    abstract fun get(): A
+
+    @Deprecated(DeprecatedUnsafeAccess, ReplaceWith("map { body(it); it }"))
+    fun onSuccess(body: (A) -> Unit): Try<A> {
+        foreach(body)
+        return this
+    }
+
+    @Deprecated(DeprecatedUnsafeAccess, ReplaceWith("fold ({ Try { body(it); it }}, { Try.pure(it) })"))
```

Comment:
Technically true, but hard to remember

Reply:
We are gonna move them to an `unsafe` syntax package if it makes sense so even if deprecated people can still use them if they want to. The `DeprecatedUnsafeAccess` constant has that in the description now with instructions to use the new package.